### PR TITLE
Release v6.0.0-beta.7

### PR DESCRIPTION
**Pull request by GretaD**

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
Can we get https://github.com/nextcloud/nextcloud-vue/pull/3165 and https://github.com/nextcloud/nextcloud-vue/pull/3157 in before a new beta release? This would really make the release a lot more usable 😉

**Comment by GretaD** [comment]
> Can we get #3165 and #3157 in before a new beta release? This would really make the release a lot more usable wink

i can do another one soon, so we're not blocked for mail, but @ChristophWurst ?

**Comment by raimund-schluessler** [comment]
> > Can we get #3165 and #3157 in before a new beta release? This would really make the release a lot more usable wink
> 
> i can do another one soon, so we're not blocked for mail, but @ChristophWurst ?

Ok, fine with me. I just thought they would be good for mail as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+### :rocket: Enhancements
+
+- Fix for border-radius being too large for 3-line entries like in Mail [\#3136](https://github.com/nextcloud/nextcloud-vue/pull/3136) ([jancborchardt](https://github.com/jancborchardt))
+- Add related resources panel component [\#3081](https://github.com/nextcloud/nextcloud-vue/pull/3081) ([Pytal](https://github.com/Pytal))
+- Add a native datetime picker [\#3063](https://github.com/nextcloud/nextcloud-vue/pull/3063) ([JuliaKirschenheuter](https://github.com/JuliaKirschenheuter))
+
+### :bug: Fixed bugs
+
+- Fix spacing in app navigation and make the setting pill style [\#3158](https://github.com/nextcloud/nextcloud-vue/pull/3158) ([juliushaertl](https://github.com/juliushaertl))
```
Comment (raimund-schluessler) [inline CHANGELOG.md:15]: This was not backported, and is not in stabel6. Although it's also not correct here afaik.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
+[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v7.0.0-beta.0...v6.0.0-beta.7)
+
+### :rocket: Enhancements
+
+- Fix for border-radius being too large for 3-line entries like in Mail [\#3136](https://github.com/nextcloud/nextcloud-vue/pull/3136) ([jancborchardt](https://github.com/jancborchardt))
+- Add related resources panel component [\#3081](https://github.com/nextcloud/nextcloud-vue/pull/3081) ([Pytal](https://github.com/Pytal))
+- Add a native datetime picker [\#3063](https://github.com/nextcloud/nextcloud-vue/pull/3063) ([JuliaKirschenheuter](https://github.com/JuliaKirschenheuter))
```
Comment (raimund-schluessler) [inline CHANGELOG.md:11]: Non of them is in stable6, I think.
The changelog generator apparently took the wrong branch.

**Comment by GretaD** [comment]
one pr is merged, so i think its fine to wait for the second one

**Comment by ChristophWurst** [comment]
I'm not sure when the second PR will land so let's do a release now and another one later

**Comment by raimund-schluessler** [comment]
> I'm not sure when the second PR will land so let's do a release now and another one later

I am just about to rebase the second one, so give me 5 min.

File: CHANGELOG.md (hunk 2 above)
Comment (GretaD) [reply CHANGELOG.md:11]: did i do smth wrong, or it is ok if i just remove these prs and release it?

**Review by ChristophWurst — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
When will this be released?

**Comment by ChristophWurst** [comment]
@GretaD is working on it

**Pull request merged**